src/Options.jsx

```diff
@@ -30,16 +31,18 @@ class Options extends React.Component {
 
     if (changeSize && Select) {
       const Option = Select.Option;
+      const defaultOption = `${props.pageSizeOptions[0]}`;
+      const options = props.pageSizeOptions.map((opt, i) => (
+        <Option key={i} value={`${opt}`}>{buildOptionText(opt)}</Option>
```

Comment:
The `Select`'s `defaultValue` propType is a string. The conversion from to number to string was to match this and thus this had to be a string too for the default value to work. 

This said, the `Select`'s `defaultValue` propType could be modified to be string OR number. What do you think?